package.json

```diff
@@ -26,6 +26,7 @@
     "postversion": "git push && git push --tags",
     "yalc:publish": "yarn build && yalc publish --push",
     "test": "jest",
+    "test:real": "jest ./src/__tests__/lmdb-real-data",
```

Comment:
again, I'm not sure if this test should be run by default (just like in the leveldb)

Reply:
It's already skipped